tools/ports/contrib/glfw3.py

```diff
@@ -45,9 +45,9 @@ def get(ports, settings, shared):
   def create(final):
     root_path = os.path.join(ports.get_dir(), 'contrib.glfw3')
     source_path = os.path.join(root_path, 'src', 'cpp')
-    source_include_paths = [os.path.join(root_path, 'external', 'GLFW'), os.path.join(root_path, 'include', 'GLFW')]
+    source_include_paths = [os.path.join(root_path, 'external'), os.path.join(root_path, 'include')]
     for source_include_path in source_include_paths:
-      ports.install_headers(source_include_path, target='GLFW')
+      ports.install_headers(source_include_path, pattern="GLFW/*.h", target='contrib.glfw3')
```

Comment:
Can you not just change target to `contrib.glfw3/GLFW` and avoid changing `install_headers` itself?

Reply:
Ironically this is what I had in my first version of this fix (see [commit](https://github.com/emscripten-core/emscripten/pull/21586/commits/b363589e6a783731d2ae89a6d233742b2e69bde1)).

I believe it does require a slight change in install_headers

```python
      dest = os.path.join(dest, *target.split('/'))
```

because I am not sure what `path.join` would do if dest is a windows path and target is `contrib.glfw3/GLFW` unless a path that mix `\` and `/` is ok with python...

Reply:
I think windows fine with `/`.  

If you are worried you can pass `target=os.path.join('contrib.glfw3', 'GLFW')`.. but I wouldn't worry.

Reply:
I reverted any changes in `install_headers` and although I think it would work on Windows, I think it is better safe than sorry.